### DISCUSSION TITLE
The GitHub Enterprise Server 3.9 release candidate is here

### DISCUSSION THREAD

**Discussion by tuves** [community/community#55068] [Enterprise]
GitHub Enterprise Server 3.9 brings new capabilities to help companies create and ship secure software, more often. Here are a few highlights:

**Autoscaling with self-hosted runners** – Actions administrators can configure auto scaling self-hosted actions runners using the actions runner controller and runner scale sets.
**The Projects public beta has seen numerous improvements** – We're continuing to add more functionality to Projects in GitHub Enterprise Server, to make planning and tracking your work in GitHub even more effective.
**REST API now has versioning** – API releases will be named with the date on which they are released (for example, 2021-04-01). Users will pick what version they want to use on a request-by-request basis, and each version will be supported for two years.
**Establishing semantic conventions for logging** – As part of GitHub's gradual migration to internal semantic conventions for [OpenTelemetry](https://opentelemetry.io/), we have updated many of the field names in our logging statements. This makes it clearer what each statement is referencing and allows you to troubleshoot problems quicker.

Read more about it [here](https://github.blog/changelog/2023-06-08-the-github-enterprise-server-3-9-release-candidate-is-available/)!

**Comment by Gso-1** [comment]
Encantado de entablar comunicación con usted pero para ello necesito que se comunique conmigo al correo gasparsalinasoyaneder@educarparaavanzar.onmicrosoft.com y si quiere una suscripción tiene un costo de 45000 mensual

**Comment by DanDevel** [comment]
That's great to hear about the new capabilities in GitHub Enterprise Server 3.9! Here's a quick summary of the highlights:

Autoscaling with Self-Hosted Runners: Actions administrators can now configure autoscaling for self-hosted actions runners using the actions runner controller and runner scale sets. This feature enables more efficient resource allocation and scaling based on demand.

Projects Public Beta Improvements: The Projects feature in GitHub Enterprise Server has received numerous enhancements. These improvements aim to enhance planning and tracking of work within GitHub, making it even more effective as a project management tool.

REST API Versioning: The REST API now includes versioning. API releases will be identified by the date they are released (e.g., 2021-04-01). Users can select the specific version they want to use on a request-by-request basis. Each version will be supported for two years, allowing for better control and compatibility with API changes.

Semantic Conventions for Logging: GitHub is gradually migrating to internal semantic conventions for OpenTelemetry. As part of this process, field names in logging statements have been updated. This enhances clarity and allows for easier troubleshooting by providing more context to each statement.

**Comment by gm3dmo** [comment]
### Changes in 3.9

Be sure to check out the changes section in the release notes:

https://docs.github.com/en/enterprise-server@3.9/admin/release-notes#3.9.0-rc.1-changes
